redis-pq-demo/README.md

```diff
@@ -0,0 +1,17 @@
+# Example run-time
+
+This is a library that builds and deploys an instance of acapy that is configured to act as a mediator, and use REDIS to as a resilient method of managing acapy's event queue. 
+
+How to see a full fledged demo, running a local ledger, and a set of agents using the mediator (with redis)  
+
+
+1. Run a redis-cluster one is provided by using `docker-compose -f docker-compose.redis.yml up`
+1. Run this configuration, `docker-compose up --build`
+
+
+If you would like to run agents that connect to this mediator, the acapy playground demo can help. 
+
+1. Run `docker-compose up` from the [demo/playground](https://github.com/hyperledger/aries-cloudagent-python/blob/main/demo/playground/)
```

Comment:
similar comment to above, way better experience if you can put in the basic/happy path commands and also point them at the readme - it does have a section about the mediator ping agents script, so you could point them there.

you should point out that these two are on different networks, so without ngrok on the mediator, these won't work without additional tweaks.

Reply:
are you able to test the new commands as I cannot use ngrok.